src/System.Runtime/tests/NSStatus/String.NSStatus.cs

```diff
@@ -0,0 +1,811 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This test is machine generated. Please see the readme.md next to the test for more information. Please do not modify this test by hand.
+
+using System;
+using Xunit;
+public class NSStatus
```

Comment:
Good question. I'm not in love with the naming. The concept I'm trying to capture is that one of the specific aims of this suite of tests will be to evaluate different .NET Standard implementations against one and other. Thus "Net Standard Status".